content/documentation/prerequisites/index.adoc

```diff
@@ -0,0 +1,48 @@
+---
+title: "Prerequisites"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation"
+---
+
+= Prerequisites
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Prerequisites Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/prerequisites/
+
+== Software Requirements
+
+To run Kiali, you're going to need two things:
+
+* A container plataform, such as link:https://www.openshift.com/[Red Hat Openshift], link:https://okd.io[OKD] or link:https://kubernetes.io[Kubernetes]
+* link:https://istio.io[Istio]
+
```

Comment:
It may not be a "hard" requirement, but I guess Jaeger should also be part of the list. It may or may not come with Istio, afaik.
Maybe we could write something like:

* Jaeger is recommended to enable tracing features. It may be installed directly with Istio.

Reply:
Maybe we should remove the links in the UI for Jaeger if there isn't ...